tool/tctl/common/desktop_command.go

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright 2021 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import (
+	"context"
+	"os"
+
+	"github.com/gravitational/kingpin"
+	"github.com/gravitational/teleport"
+	"github.com/gravitational/trace"
+
+	"github.com/gravitational/teleport/lib/auth"
+	"github.com/gravitational/teleport/lib/service"
+)
+
+// DesktopCommand implements "tctl desktop" group of commands.
+type DesktopCommand struct {
+	config *service.Config
+
+	// format is the output format (text or yaml)
+	format string
+
+	// desktopList implements the "tctl desktop ls" subcommand.
+	desktopList *kingpin.CmdClause
+}
+
+// Initialize allows DesktopCommand to plug itself into the CLI parser
+func (c *DesktopCommand) Initialize(app *kingpin.Application, config *service.Config) {
+	c.config = config
+
+	desktop := app.Command("desktop", "Operate on registered desktops.")
```

Comment:
In most cases, we're referring to these resources as "Windows Desktops" as desktop access was designed with the idea that we may support other OS in the future.

That said, `tctl windows_desktop ls` is kind of a pain to type, so maybe we just make it `desktops ls` and add an `OS` column in the future if we support other desktops.